src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/TextInterfaceToListeners.java

```diff
@@ -15,7 +15,7 @@
 	 * 
 	 */
 	private static final long serialVersionUID = 7996257740483513358L;
-	private JTextField commandLine = new JTextField(60);
+	private JTextField commandLine = new JTextField(30);
```

Comment:
60 is the maximum character limit to send to a mega2560 in a single 'send'.  This was a carefully chosen number :)

Reply:
Ok ! I'll revert it, didn't know :)